README.md

```diff
@@ -26,59 +26,87 @@ Unsure what to contribute? Check out the `good first issue` [tagged Github issue
   - [Engines](#engines)
   - [Haxe/HaxeFlixel Resources](#haxehaxeflixel-resources)
   - [Visual Studio Code](#visual-studio-code)
+- [Charting](#charting)
 - [Modding](#modding)
+  - [Modding Tutorials](#modding-tutorials)
+- [Bonus](#bonus)
```

Comment:
I feel like there shouldn't be a `bonus` category, and things should just have a proper place to go 🙇‍♂️

`Engine Documentation` can be a subsection of `Engines` category
the resources from `Other Resources` is currently just a compile guide for FNF 0.2.7.1, which can be placed into the `Programming` section

Reply:
Will revert changes now.

Reply:
Done!